project_liberation/system_check.py

```diff
@@ -26,3 +33,23 @@ def check_google_api_key(app_configs: Any, **kwargs: Any) -> list:  # pylint: di
         return [GOOGLE_API_KEYS_ERRORS["length"]]
 
     return []
+
+
+def check_custom_storage_directories(setting_name: str) -> list:
+    if not hasattr(settings, setting_name):
+        return [setting_not_declared_error(setting_name)]
+
+    if not isinstance(settings.__getattr__(setting_name), str):
```

Comment:
Please change it into `getattr` function

Reply:
Done